CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
 # CHANGELOG
 
-TBD
\ No newline at end of file
+
```

Comment:
`git-chglog --next-tag v0.1.0 > CHANGELOG.md`

the version 0.0.0 is not a valid version

Reply:
Fixed.